tools/update_kernel_commits.py

```diff
@@ -293,11 +332,11 @@ def get_kernel_tags_from_dockerhub(username, repository, search_pattern: str = N
     regex_pattern = None
 
     # convert search patterns to gerexp
-    if search_pattern:
+    if search_pattern=="":
```

Comment:
This means that when no search pattern is provided, the regex will be created from "" (which becomes ^$ after escaping). That regex only matches empty strings, so it might unintentionally filter out all tags.